official/resnet/keras/keras_imagenet_benchmark.py

```diff
@@ -477,7 +555,23 @@ def benchmark_xla_8_gpu_fp16_tweaked(self):
     FLAGS.model_dir = self._get_model_dir('benchmark_xla_8_gpu_fp16_tweaked')
     FLAGS.batch_size = 256 * 8  # 8 GPUs
     # FLAGS.tf_gpu_thread_mode = 'gpu_private'
-    FLAGS.data_prefetch_with_slack = True
+    FLAGS.data_delay_prefetch = True
+    self._run_and_report_benchmark()
+
+  def benchmark_xla_8_gpu_fp16_tfdata_exp(self):
```

Comment:
Since this is temporary until proven, maybe consider:
`benchmark_xla_8_gpu_fp16_slack`

Not a big deal and I can see reasons for your naming as well.